src/adapters/adapter.moment.js

```diff
@@ -2,7 +2,7 @@
 
 'use strict';
 
-var moment = require('moment');
```

Comment:
I think we'll move this file to an external repo and make it optional, but until then won't you need this line or else you'll get errors about `moment` not being defined? you'll also need the `optional` rollup plugin

Reply:
rollup & terser will just treat moment as global and not touch it and it will works just fine. which is the way it works now when you include moment ahead of loading Chart.js in the browser.